docs/docs/cloud/cloud-cli.mdx

```diff
@@ -48,6 +48,90 @@ meltano cloud config env delete TAP_GITHUB_AUTH_TOKEN
 
 See the [reserved variables](/cloud/platform#reserved-variables) docs for more details on variables that are reserved for use by Meltano Cloud.
 
+### `notification`
+
+The `notification` command provides a way for managing your [Meltano Cloud notifications](concepts#meltano-cloud-notifications).
+
+#### `list`
+
+List all notifications configured for your current default project or organization.
+
+```sh
+$ meltano cloud config notification list
+╭─────────────────────┬─────────┬──────────╮
+│ Recipient           │ Type    │ Status   │
+├─────────────────────┼─────────┼──────────┤
+│ https://melty       │ webhook │ active   │
+│ dragon@meltano.com  │ email   │ active   │
+╰─────────────────────┴─────────┴──────────╯
+```
+
+#### `set`
+
+Set a new notification or override an existing one.
+
+##### Case-Sensitivity in Values
+
+Take note that all the values in the commands and configuration settings are case-sensitive. This means job_created, Job_Created, and JOB_CREATED would be considered as different events. Please ensure to use the correct casing to avoid configuration issues or unexpected behaviors.
+
+
+Interactive mode
+```sh
+meltano cloud config notification set <type>
+```
+
+Non-interactive mode:
+```sh
+meltano cloud config notification set <type> --recipient <target_recipient> --filter <filter>
+````
+
+**`options`**
+
+`--recipient` (Required): The recipient to send the notification to.
+
+`--filter`: An optional filter you want to check against, you can supply multiple filters which will all be joined as an OR expression.
+Values within the filter are validated as an AND Expression.
+
+```sh
+meltano cloud config notification set webhook --recipient http://test+melty --filter '{"events": ["job_created", "job_succeeded"], "status": ["succeeded"]}' --filter '{ "status": ["failed"]}'
+```
+The above example sets a webhook notification to send to ﻿http://test+melty if the event is ﻿(job_created OR job_succeeded AND status = succeeded) OR status = failed.
```

Comment:
Can we have a few examples of what we think will be the most common filters? 

* job success
* job failure

any others?

Reply:
I was using general examples in order to showcase how the logic works, we only support a single job_run event right now but that can be changed. I did this because it didn't make a lot of sense to split up into job_run_succeeded, job_run_cancelled when we were setting the "status" as a filter option.

I almost wonder if we should split it into multiple events and not have "status" as a filter, but instead derive the "status" from the event. So instead of 
events: ["job_run"], status: ["succeeded", "failed"]

We had
events: ["job_run_succeeded", "job_run_failed"]

Initially I went with the first option to make it easier to specify the same statuses for different events so I could do something like `post this notification for all "job_run" and "project_create" that "succeeded" or "failed"` instead of
events: ["job_run_succeeded", "job_run_failed", "project_create_succeeded", "project_create_failed"]

This is also given the assumption that most of our events that trigger notifications will have a "status".

Reply:
Let's not change the model - I think it makes sense as is. All I'm looking for is an example of what folks would set (assuming most people like to copy paste things)

### Job Failures

To subscribe only to job failures you would set the filter to: 

```
example command
```

----

Though now that you mention it, this paragraph says we have the following events `job_created", "job_succeeded` but that doesn't match the filter listed here https://deploy-preview-7968--meltano.netlify.app/cloud/concepts/#supported-notification-filters